articles/legacy/migration-guides/management-api-v1.md

```diff
@@ -0,0 +1,258 @@
+
+This document describes the major differences between Auth0's Management API v1 and Management API v2, and details how to migrate your implementation to the latest API.
+
+## Am I affected by this?
+
+You are impacted if you are using any of the [Management API v1](/api/management/v1) endpoints.
+
+## What should I do?
+
+If your implementation uses any of the [Management API v1](/api/management/v1) endpoints, you have to update it in order to invoke the matching [Management API v2](/api/management/v2) endpoint. Before you make any changes refer to the [Endpoints Mapping Matrix](#endpoints-mapping-matrix) to find the suitable alternative for your case.
+
+There are also some other more generic changes. The most prominent one is that the authentication mechanism for the API changes. This means that you have to change how you get a token in order to authenticate with the API and access any of its endpoints. For more information refer to [Update the API Authentication Mechanism](#update-the-api-authentication-mechanism).
+
+Other generic changes include:
+
+- [The user metadata structure](#user-metadata-structure)
+- [Connections-related changes](#connections-related-changes)
+- [Consolidation of endpoints](#consolidation-of-endpoints)
+- [Input validation and error messages](#input-validation-and-error-messages)
+- [How `null` values in `PATCH` calls are handled](#update-fields-with-null-values)
+- [All endpoints require IDs](#all-endpoints-require-ids)
+
+### Update the API Authentication Mechanism
+
+Management API v1 requires sending an `access_token` obtained by performing a `POST /oauth/token` request along with the **Client Id** and **Client Secret**. All subsequent requests must include this `access_token` in the `Authorization` header: `Authorization: Bearer {access_token}`.
+
+Management API v2 lets you issue a [JWT](/jwt) token with specific granted permissions (known as scopes), and is signed with a client API key and secret for the entire tenant.
+
+For example, the [GET /api/v2/clients](/api/management/v2#!/Clients/get_clients) (which retrieves a list of all the client applications defined in the [Dashboard](${manage_url})) requires two scopes: 
+
+- `read:clients`, and 
+- `read:client_keys`
+
+These are scopes that your `access_token` should include. If, however, you try to use the same token for the [POST /api/v2/clients](/api/management/v2#!/Clients/post_clients) (which creates a new client) you will get an authorization error. The reason is that that endpoint requires the `create:clients` scope which your `access_token` did not include.
+
+All requests to the API v2 must include the `access_token` in the `Authorization` header: `Authorization: Bearer {access_token}`.
+
+What you should do here is migrate your implementation to use the new tokens. For more information on how to generate and use such a token refer to [The Auth0 Management APIv2 Token](/api/management/v2/tokens).
+
+#### The id_token and special scopes
```

Comment:
This will also be deprecated in Q2 2018 (`Using ID Tokens for Management API Authentication`). I'm thinking of adding a warning in this section saying so.